frontend/src/configs/common.js

```diff
@@ -0,0 +1,6 @@
+const PLAUSIBLE_DATA_DOMAIN_STAGING =
```

Comment:
would we expect `common.js` file to hold all future strings as consts used across our codebase?

Is it a better idea to define this file to be more specific to just Plausible consts? I think it's fine for now but in the future, if this file becomes larger, we could consider splitting up more relevant consts.

Reply:
Thanks for the great question, Dimple!

Since the file is inside `src/configs/`, the intention for `common.js` is only for hosting commonly used stuff within this directory

For files used across the whole codebase, we have a different `common` directory for it in `src/common/`, following [Dependency Inversion](https://en.wikipedia.org/wiki/Dependency_inversion_principle#Implementations) principle in SOLID!

Hope this makes sense 🙆‍♂️ Thank you!

Reply:
thanks for the explanation! That makes sense.